apps/growthbook/next.config.js

```diff
@@ -0,0 +1,10 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  basePath: '',
+  assetPrefix: '',
```

Comment:
```suggestion
  assetPrefix: './',
```
Looks like this was the change needed for our infrastructure to correctly serve the static assets. Let me know if this causes any issues on your end